drivers/dma/dma_stm32.c

```diff
@@ -632,7 +774,11 @@ DMA_STM32_EXPORT_API int dma_stm32_get_status(const struct device *dev,
 	}
 
 	stream = &config->streams[id];
+#if defined(CONFIG_DMA_STM32_V3)
+	stat->pending_length = LL_DMA_GetBlkDataLength(dma, dma_stm32_id_to_stream(id));
+#else
 	stat->pending_length = LL_DMA_GetDataLength(dma, dma_stm32_id_to_stream(id));
```

Comment:
ditto

Reply:
changed with a redefiniion of the LL_DMA_GetBlkDataLength for the stm32U5x, in the header file